pkg/controller/master/supportbundle/manager.go

```diff
@@ -0,0 +1,207 @@
+package supportbundle
+
+import (
+	"errors"
+	"fmt"
+
+	harvesterv1 "github.com/harvester/harvester/pkg/apis/harvesterhci.io/v1beta1"
+	ctlappsv1 "github.com/rancher/wrangler/pkg/generated/controllers/apps/v1"
+	ctlcorev1 "github.com/rancher/wrangler/pkg/generated/controllers/core/v1"
+	"github.com/sirupsen/logrus"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+
+	"github.com/harvester/harvester/pkg/version"
+)
+
+type Manager struct {
+	deployments ctlappsv1.DeploymentClient
+	nodeCache   ctlcorev1.NodeCache
+	services    ctlcorev1.ServiceClient
+}
+
+func (m *Manager) getManagerName(sb *harvesterv1.SupportBundle) string {
+	return fmt.Sprintf("supportbundle-manager-%s", sb.Name)
+}
+
+func (m *Manager) Create(sb *harvesterv1.SupportBundle, image string) error {
+	deployName := m.getManagerName(sb)
+	logrus.Debugf("creating deployment %s with image %s", deployName, image)
+
+	serviceAccountName, err := m.getServiceAccountName(sb.Namespace)
```

Comment:
I think we can use the fixed SA name "harvester"

Reply:
fixed.